pypeerassets/provider/holytransaction.py

```diff
@@ -43,7 +43,7 @@ def is_testnet(self):
     def network_long_to_short(name):
         """convert long network name like "peercoin" to "ppc"."""
 
-        if len(name) > 4:
+        if len(name) > 3:
```

Comment:
I believe it would be logical to do without this *if* statement since we are checking directly if the input matches one of the strings 'peercoin' and 'peercoin-testnet'. Other than that I can imagine simply querying networks, https://github.com/PeerAssets/pypeerassets/blob/master/pypeerassets/networks.py#L22 , and pulling the respective long or short name would be best.